ocs/ocp.py

```diff
@@ -283,3 +284,19 @@ def exec_ceph_cmd(ceph_cmd):
     if isinstance(out, list):
         return [item.toDict() for item in out if item]
     return out.toDict()
+
+
+def getbase64_ceph_secret(keyring_name):
+    """
+    Get ceph keyring in base64 encoded format.
+
+    Args:
+        keyring_name (str): Keyring name i.e. client.admin, client.kubernetes
```

Comment:
Could you please add more explanation here?
Where are we getting the client from?

Reply:
@RazTamir, updated the args comments section.